tests/unit/local/apigw/test_local_apigw_service.py

```diff
@@ -341,6 +341,15 @@ def test_status_code_not_int(self):
                                                    binary_types=[],
                                                    flask_request=Mock())
 
+    def test_status_code_int_str(self):
```

Comment:
Can you add a test for when ValueError is raised when status_code is not a positive integer as a string?

Reply:
Sure thing. Added.